packages/plugin-user-agent-enrichment-browser/src/typings/user-agent-enrichment-plugin.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable @typescript-eslint/no-empty-interface */
+import { EnrichmentPlugin } from '@amplitude/analytics-types';
+
+export interface Options {
+  osName?: boolean;
+  osVersion?: boolean;
+  deviceManufacturer?: boolean;
+  deviceModel?: boolean;
+}
+
+export interface CreateUserAgentEnrichmentPlugin {
```

Comment:
nits: can we use `UserAgentEnrichmentPlugin` to align with the naming convention of this repo?

Reply:
The interface is starting with `Create` in other packages.
Like `CreatePageViewTrackingPlugin`, `CreateWebAttributionPlugin`, etc.